src/shared/varlink.h

```diff
@@ -92,6 +92,8 @@ int varlink_call_full(Varlink *v, const char *method, JsonVariant *parameters, J
 static inline int varlink_call(Varlink *v, const char *method, JsonVariant *parameters, JsonVariant **ret_parameters, const char **ret_error_id) {
         return varlink_call_full(v, method, parameters, ret_parameters, ret_error_id, NULL);
 }
+int varlink_call_and_log(Varlink *v, const char *method, JsonVariant *parameters, JsonVariant **ret_parameters, const char **ret_error_id);
```

Comment:
i'd use the "xyz_verbose()" naming scheme for functions that generate log messages about what they do. i.e. mount_verbose() is more about making traceable in the logs which mounts are established. It *also* generates a lot message on error, but that's just a secondary facet I'd say.

the various xyz_and_log() and xyz_and_warn() functions we have in the tree otoh are really about geneerating error messages however, i.e they do not add descriptive logging to some already existing syscall or so like mount_verbose() is doing it. 

hence this here seems to be a case for the xyz_and_log() suffix, since it really just adds error logging, it doesn't add descriptive logging about what it does so (the varlink layer does that anyway)